src/app/CodeRepo/lib/backends/github/github_event_emitter.js

```diff
@@ -10,55 +11,51 @@ class GithubEventEmitter extends AsyncEventEmitter {
         super();
     }
 
-    async testEventStream() {
-        // Should trigger a ping
-    }
-
     async _setupWebServer(port) {
-        this.app = new Koa();
+        const app = new Koa();
 
-        this.app.use(bodyParser());
+        app.use(bodyParser());
 
-        this.app.use(async (ctx) => {
+        app.use(async (ctx) => {
             const body = ctx.request.body;
             if (body && body.zen) {
                 this.emit("ping", body);
             } else if (body && body.pull_request) {
                 if (body.action && body.action === "opened") {
                     this.emit("pull-request-open", body);
                 } else {
-                    console.log("unknown pull-request event received");
+                    ServiceMgr.instance.log("verbose", "unknown pull-request event received");
                 }
             } else {
-                console.log("unknown event received");
+                ServiceMgr.instance.log("verbose", "unknown event received");
+                ServiceMgr.instance.log("debug", body);
             }
 
             ctx.response.status = 200;
         });
 
-        const server = this.app.listen(port);
+        this.server = app.listen(port);
 
         return new Promise((resolve, reject) => {
-            server.on("listening", () => {
-                server.removeListener("error", reject);
+            this.server.on("listening", () => {
+                this.server.removeListener("error", reject);
                 resolve(`Webhook server up on port ${port}`);
             });
-            server.once("error", reject);
+            this.server.once("error", reject);
         });
     }
 
     async teardownWebServer() {
-        console.log("Tearing down web hook server");
-        this.app.close();
+        ServiceMgr.instance.log("verbose", "Tearing down web hook server");
+        this.server.close();
```

Comment:
there is no function close on app, I couldnt find anything in the examples that indicates the app should be closed